project-reports/2024/2024-Q2-Hyperledger-Identus.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: 2024 Q2 Hyperledger Identus
+parent: 2024
+grand_parent: Project Updates
+nav_exclude: true
+---
+
+Created by Fayyaadh Adams with input from other maintainers
+
+
+# Project Health
+
+The project is fairly new to the Hyperledger Ecosystem however we continue to have steady contributors. We are currently not on the LFX dashboard yet and are waiting to be added to gain further insights. In the last three weeks alone, we have seen 3 new contributors (not code related) show up and additional usage of our code within production environments (https://socious.io/ - production use case). 
+
+
+# Questions/Issues for the TOC
+
+Is there a way to access funds for open source contirbutions and development? 
+
+
+# Releases
+
+  * Identus V2.12
+    * Identus Cloud Agent - V.1.33.0
+    * Identus Mediator - V.0.14.2
+    * Identus Apollo - V.1.2.14
+    * Identus Edge Agent SDK TS - V.5.1.0
+    * Identus Edge Agent SDK KMP - V.3.0.0
+    * Identus Edge Agent SDK SWIFT - V.6.0.0
+    * Identus Docs - V.1.92.0
+
+
+
+# Overall Activity in the Past Quarter
+
+The main focus this quarter, and continuing into the next, is to achieve a rich set of features within Identus. The contribution of the code base from IOG to Hyperledger was completed in early July 2024. Initially it was only the cloud agent respository that was part of Hyperledger that started out as the Open Enterprise Agent Lab. The rest of the repositories were public repositories in IOG and has since been migrated to Hyperledger. 
+
+The mailing list is not the primary tool for discussions, instead we encourage discussions and questions within the relevant Discord channels. To enhance support for community members, we have enlisted the CommunityOne bot, which is trained on the Identus documentation. This has been extremely helpful for developers seeking immediate answers when a contributor or maintainer is unavailable. The majority of development now is to get to feature complete and fix exisiting bugs that get reported by the community. 
```

Comment:
It would be great to learn what it took to train the CommunityOne bot and any recommendations you might have for other projects who are interested in doing something similar.

Reply:
Is this enabled on Hyperledger's Discord or is it run on the Atala's?

Reply:
This is the specific bot we use https://communityone.io/

Reply:
It is enabled on both Discord servers

Reply:
> It was actually a very simple. I would highly recommend getting documentation of installation, setup and general FAQ's together. The bot is then fed a .txt file and deployed to a particular channel. I would be happy to show others how this works :)

It would be great to have you present at a future TOC meeting.

Reply:
I would love to do so